src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/TraceLoggingEventSource.cs

```diff
@@ -151,6 +152,10 @@ public unsafe void Write(string? eventName, EventSourceOptions options)
         /// public instance properties of data will be written recursively to
         /// create the fields of the event.
         /// </param>
+#if !ES_BUILD_STANDALONE
+        [RequiresUnreferencedCode("EventSource Write Event is not safe due to type description data. " +
```

Comment:
Extract the message into a common const string, so it can be shared across all the methods.